### ISSUE TITLE
how to use bone length map?

### ISSUE THREAD

**Issue by guker** [hwangdonghyun/VNect-TensorFlow-Keras#3]
hello, I recently study vnect, and find  that your code do not use bone length information as  intermediate supervision. However，paper mention this bone length as intermediate supervison,
so I  am confused.

![image](https://user-images.githubusercontent.com/4830247/82996996-5f8e5e80-a038-11ea-968b-d333924b8796.png)

**Comment by hwangdonghyun** [comment]
I guess you don't have to give a loss as the common supervised learning. It's just intermediate cue to estimate accurate 3d pose.
